Source/WarpX.H

```diff
@@ -1035,17 +1035,43 @@ private:
     amrex::Vector<std::array< std::unique_ptr<amrex::MultiFab>, 3 > > Efield_avg_fp;
     amrex::Vector<std::array< std::unique_ptr<amrex::MultiFab>, 3 > > Bfield_avg_fp;
 
-    //EB grid info
-    amrex::Vector<std::array< std::unique_ptr<amrex::MultiFab>, 3 > > Venl;
+    // EB: Lengths of the mesh edges
     amrex::Vector<std::array< std::unique_ptr<amrex::MultiFab>, 3 > > m_edge_lengths;
+    // EB: Areas of the mesh faces
     amrex::Vector<std::array< std::unique_ptr<amrex::MultiFab>, 3 > > m_face_areas;
+
+    // m_flag_info_face, m_flag_ext_face, m_area_mod, m_borrowing, ECTRhofield and Venl are only
+    // needed for the ECT solver
+
+    // EB: for every mesh face flag_info_face contains a:
+    //          * 0 if the face needs to be extended
+    //          * 1 if the face is large enough to lend area to other faces
+    //          * 2 if the face is actually intruded by other face
+    // It is initialized in WarpX::MarkCells
```

Comment:
Doxygen multi-line comments:
```suggestion
    /** EB: for every mesh face flag_info_face contains a:
     *          * 0 if the face needs to be extended
     *          * 1 if the face is large enough to lend area to other faces
     *          * 2 if the face is actually intruded by other face
     * It is initialized in WarpX::MarkCells and only used for the ECT solver */
```